abi3info/__init__.py

```diff
@@ -2,7 +2,7 @@
 The abi3info APIs.
 """
 
-from typing import Final
+from typing import Final, Dict
```

Comment:
FWIW we can do `from __future__ import annotations` here and elsewhere to continue to use the `dict[...]` style 🙂

Reply:
I'll do that after merge though, since it's tiny!